zip-0208.rst

```diff
@@ -70,14 +73,12 @@ to the list of integer constants.
 
 In section 5.3 (Constants), define PostBlossomPoWTargetSpacing := 75 seconds.
 
-TODO: check that PostBlossomPoWTargetSpacing is reasonable.
```

Comment:
So we are going with 75 seconds? Or are there still plans to test this at a lower value?

Reply:
My impression from previous engineering meetings was that we'd decided on 75s. I'm happy to do more testing.